Xamarin.Forms.Platform.GTK/Controls/ActivityIndicator.cs

```diff
@@ -170,29 +31,21 @@ public void Stop()
 			QueueDraw();
 		}
 
-		protected override bool OnExposeEvent(EventExpose evnt)
+		public void UpdateColor(Color color)
 		{
-			base.OnExposeEvent(evnt);
-
-			using (var cr = CairoHelper.Create(GdkWindow))
+			if (color.IsDefault)
 			{
-				cr.Rectangle(
-					evnt.Area.X,
-					evnt.Area.Y,
-					Height,
-					Width);
-
-				cr.Clip();
-
-				Draw(cr);
+				_color = new Color(0, 0, 0);
```

Comment:
In other platforms, the default colors is different. You can do something like:

`Color.Default.ToGtkColor()`